composer.json

```diff
@@ -25,11 +25,12 @@
         }
     },
     "require" : {
-        "php"                : ">=5.3.3"
+        "php"                : ">=5.4"
```

Comment:
As GuzzleHttp v4+ requires PHP 5.4, is this ok?
It will break previous applications...